coordinator/src/main/java/org/apache/uniffle/coordinator/SimpleClusterManager.java

```diff
@@ -64,6 +64,14 @@ public class SimpleClusterManager implements ClusterManager {
   private final Map<String, ServerNode> servers = JavaUtils.newConcurrentMap();
   private final Cache<ServerNode, ShuffleServerInternalGrpcClient> clientCache;
   private Set<String> excludeNodes = Sets.newConcurrentHashSet();
+  /**
+   * ServerNode whose heartbeat is lost
+   */
+  Set<ServerNode> lostNodes = Sets.newHashSet();
```

Comment:
When do we clean this set? Will it cause memory leak if we don't restart server for a long time?

Reply:
OK.